src/modules/icons/index.js

```diff
@@ -11,4 +11,4 @@ export { default as Tags } from './tags.svg';
 export { default as Sharing } from './sharing.svg';
 export { default as Venue } from './venue.svg';
 export { default as Website } from './website.svg';
-
+export { default as EventsListIcon } from './events-list.svg';
```

Comment:
You can just name this `EventsList`, all the other icons in this file don't have the postfix of `Icon`

Reply:
I added the suffix to differentiate it from the Edit component [here](https://github.com/the-events-calendar/the-events-calendar/pull/3681/files#diff-ed3436a73ae429cb6ac1a748f3606de0a76b8edc03896463dba4018493ede7adR9) which used the `EventsList` label.

Reply:
this is updated